src/Microsoft.Framework.PackageManager/Restore/NuGet/NuGetv2Feed.cs

```diff
@@ -17,13 +17,19 @@ namespace Microsoft.Framework.PackageManager.Restore.NuGet
     public class NuGetv2Feed : IPackageFeed
     {
         private static readonly XNamespace _defaultNamespace = XNamespace.Get("http://www.w3.org/2005/Atom");
-        private static readonly XName _xnameEntry = XName.Get("entry", _defaultNamespace.NamespaceName);
-        private static readonly XName _xnameTitle = XName.Get("title", _defaultNamespace.NamespaceName);
-        private static readonly XName _xnameContent = XName.Get("content", _defaultNamespace.NamespaceName);
-        private static readonly XName _xnameLink = XName.Get("link", _defaultNamespace.NamespaceName);
-        private static readonly XName _xnameProperties = XName.Get("properties", "http://schemas.microsoft.com/ado/2007/08/dataservices/metadata");
-        private static readonly XName _xnameId = XName.Get("Id", "http://schemas.microsoft.com/ado/2007/08/dataservices");
-        private static readonly XName _xnameVersion = XName.Get("Version", "http://schemas.microsoft.com/ado/2007/08/dataservices");
+        private static readonly XNamespace _odataMetadataNamespace = XNamespace.Get("http://schemas.microsoft.com/ado/2007/08/dataservices/metadata");
```

Comment:
nit: you don't need `XNamespace.Get` - just 
```
 private static readonly XNamespace _odataMetadataNamespace = "http://schemas.microsoft.com/ado/2007/08/dataservices/metadata"; 
```
it will know to convert the string to `XNamespace`